### PR TITLE
Add pylops

### PR DESCRIPTION
**Pull request by jan-janssen**
Currently available teams are:
- python @conda-forge/help-python

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams.
Consider asking on our [Gitter channel](https://gitter.im/conda-forge/conda-forge.github.io)
if your recipe isn't reviewed promptly.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
` scikit-fmm -> python=3.6 ` - so I guess we need to build `scikit-fmm` first.

File: recipes/scikit-fmm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - skfmm
+
+about:
+  home: https://github.com/scikit-fmm/scikit-fmm
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'scikit-fmm is a Python extension module which implements the fast marching method.'
+
+  # The remaining entries in this section are optional, but recommended.
```
Comment (chrisburr) [inline recipes/scikit-fmm/meta.yaml]:
```suggestion
```

File: recipes/scikit-fmm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy
```
Comment (chrisburr) [inline recipes/scikit-fmm/meta.yaml]: I assume this uses the Numpy C API if so: http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy

If not you probably don't need `numpy` here.

File: recipes/pylops/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    - scipy
+
+test:
+  imports:
+    - pylops
+
+about:
+  home: https://github.com/equinor/pylops
+  license: LGPL
+  license_family: LGPL
```
Comment (chrisburr) [inline recipes/pylops/meta.yaml]:
```suggestion
  license_family: GPL
```

File: recipes/pylops/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    - scikit-fmm
+    - scipy
+
+test:
+  imports:
+    - pylops
+
+about:
+  home: https://github.com/equinor/pylops
+  license: LGPL
```
Comment (chrisburr) [inline recipes/pylops/meta.yaml]:
```suggestion
  license: LGPL-3.0
```

File: recipes/pylops/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  build:
+      - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - llvmlite
```
Comment (chrisburr) [inline recipes/pylops/meta.yaml]: Are these all needed in `host`? (Same comment about the Numpy C API as below.)

File: recipes/scikit-fmm/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipes/scikit-fmm/meta.yaml]: I tired this in https://github.com/conda-forge/staged-recipes/pull/8362/commits/6e2d2276e90676e6ecebbef656ebf40c33ae7a9c - but then the setup fails.

**Comment by jan-janssen** [comment]
@chrisburr thanks a lot for your review - I included the changes.

File: recipes/scikit-fmm/meta.yaml (hunk 2 above)
Comment (chrisburr) [reply recipes/scikit-fmm/meta.yaml]: Then you're probably using the C API and should use `pin_compatible` in `run` (see above link for details).

Comment (jan-janssen) [reply recipes/scikit-fmm/meta.yaml]: As far as I see it, it is only importing `from numpy.distutils.core import setup` I do not think it is necessary to pin then, or?

**Comment by mrava87** [comment]
Hi,
Nice to see that you are trying to add pylops to conda-forge, that’s amazing ;)

I am not into the details of this process and how much effort would it take you to build scikit-fmm, but this is just a very mild optional dependency for Pylops. It is used by a single operator, which can still work in the ‘byot’=bring-your-own-table mode, allowing users to use traveltime curves that are computed with any other python or external tool. Hope this helps, and thanks a lot again

**Comment by mrava87** [comment]
Ops, realized I was too slow, you added scikit-fmm :) fantastic 👍

**Comment by jan-janssen** [comment]
@mrava87 Sure, should I add you as a maintainer?

**Comment by mrava87** [comment]
Sure, you can do that :)

**Pull request merged**